docs/recipes/client-server/upload-file-from-client.md

```diff
@@ -0,0 +1,90 @@
+# How do I upload a file from the client?
+[Fable](https://fable.io/) makes it quick and easy to upload files from the client. Both the standard and the minimal template comes with Fable support by default.
+
+---
+
+#### 1. Create a File
+
+Create a file in the client project named `FileUpload.fs` somewhere before the `Index.fs` file and insert the following:
+
+```fsharp
+module FileUpload
+
+open Fable.React
+open Fable.React.Props
+open Fable.FontAwesome
+open Fable.Core
+open Fable.Core.JsInterop
+open Fulma // Only if using the standard template
+```
+
+#### 2. JavaScript Interop
+
+Insert the following lines after the *open* statements. This function will be used for JavaScript interop. [The Emit attribute](https://fable.io/docs/communicate/js-from-fable.html#Emit-when-F-is-not-enough) means that the calls to this function will be replaced by inlined JavaScript code.
+
+```fsharp
+[<Emit("new Uint8Array($0)")>]
+let createByteArray arrBuff : byte[] = jsNative
```

Comment:
is this necessary? [fable docs](https://fable.io/fable-doc/communicate/numbers.html) say `Uint8Array` should be mapped to `byte[]` automatically?

Reply:
`byte[]` *is* `Uint8Array` so you can just unbox

Reply:
I just got rid of step two and replaced the use of `createByteArray` function with `unbox` in step three.